utils/misc/ewma.go

```diff
@@ -0,0 +1,138 @@
+package misc
+
+import "sync"
+
+const (
+	// By default, we average over a one-minute period, which means the average
+	// age of the metrics in the period is 30 seconds.
+	AVG_METRIC_AGE float64 = 30.0
```

Comment:
should we be making these variables configurable ? or go ahead with the defaults